### ISSUE TITLE
Can't use precompiled gitless on some "old" linux distro

### ISSUE THREAD

**Issue by chmike** [sdg-mit/gitless#105]
For some work I'm constrained to use an old linux distribution. 
I downloaded gl-v0.8.4-linux-x86_64. Unfortunately I can't run it for some weird reason.
This is the output of `uname -a` : `Linux <hostname> 2.6.32-642.6.2.el6.x86_64 #1 SMP Wed Oct 26 08:22:04 CEST 2016 x86_64 x86_64 x86_64 GNU/Linux`. It's scientific linux 6. 

Here is the output I get:
```
$ ldd ~/bin/gl
	linux-vdso.so.1 =>  (0x00007fff951f5000)
	libdl.so.2 => /lib64/libdl.so.2 (0x00007f85660e5000)
	libz.so.1 => /lib64/libz.so.1 (0x00007f8565ecf000)
	libc.so.6 => /lib64/libc.so.6 (0x00007f8565b3a000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f8566319000)
$ gl status
gl: /lib64/libc.so.6: version `GLIBC_2.14' not found (required by /tmp/cmeessen/_MEIh5NRYj/libz.so.1)
```
Is there anything I could do to fix this problem, beside changing my job ?

**Comment by spderosso** [comment]
Yeah, sorry about this. Binary releases won't work in old distros. I believe it is because I use a fairly recent Ubuntu VM to build the linux release, and the binary that pyinstaller creates is not retro-compatible with old linux releases.

I'd have to look into to see if there's something I could do from my side so that the binary releases  work in old(er) linux distros.

As a workaround you can build gitless from source ([instructions])(https://github.com/sdg-mit/gitless#install)
